packages/plugin/src/generators/app/templates/manifest.json

```diff
@@ -7,7 +7,7 @@
             "name": "Sample widget",
             "description": "Description of the Sample Widget",
             "version": "1.0.0",
-            "src": "src/widgets/Widget",
+            "src": "src/widgets/SampleWidget",
```

Comment:
This change itself is fine, though with generate-manifest, we wouldn't include this file at all anymore right?  It probably should be added to .gitignore, or better yet, generate-manifest should just generate the manifest directly in the `dist` directory as a build artifact instead of in the source tree and then relying on CopyWebpackPlugin to copy it to the build output.